src/flows/context/query.tsx

```diff
@@ -41,44 +42,6 @@ export const QueryContext = React.createContext<QueryContextType>(
 
 const PREVIOUS_REGEXP = /__PREVIOUS_RESULT__/g
 
-const findOrgID = (text, buckets) => {
```

Comment:
moved this to a central locale to be used for the metric selector as well

Reply:
why not use the query context instead?

Reply:
Do you man to use the `query` from QueryContext in order to pass the query in and run it?